README.md

```diff
@@ -58,7 +66,27 @@ vcf_validator -i /path/to/file.vcf -r database -o /path/to/write/report/
 vcf_debugulator -i /path/to/file.vcf -e /path/to/write/report/vcf.errors.timestamp.db -o /path/to/fixed.vcf 2>debugulator_log.txt
 ```
 
-## Static build (Docker-based)
+
+### Dependencies
```

Comment:
is it ok to keep docker outside the Linux ? or it would be fine to keep it inside linux as @cyenyxe suggested in slack.

Reply:
technically docker has to be inside a linux, but I think this structure in our readme is less complex and equally understandable (as it focuses on which platform is the one doing the build, not the one the user uses).

Reply:
I'm fine with keeping Docker separate if that looks better. Maybe I would list Docker first because that is the simplest approach and users should favor that one when available.

Reply:
@cyenyxe  @jmmut  should be keep compile as subsection in every platform. I think our cmake is quite smart now to detect platform so our compile section will almost be similar in all of these.